apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -456,19 +455,16 @@ private List<RifRecordLoadResult> process(
           if (rifRecordEvent.getRecordAction().equals(RecordAction.INSERT)) {
             loadAction = LoadAction.INSERTED;
 
-            // Log if we have a non-2022 enrollment year INSERT
-            if (isBackdatedBene(rifRecordEvent)) {
-              Beneficiary bene = (Beneficiary) rifRecordEvent.getRecord();
-              LOGGER.info(
-                  "Inserted beneficiary with non-2022 enrollment year (beneficiaryId={})",
-                  bene.getBeneficiaryId());
+            if (!isSyntheticData) {
+              // don't care about return status here...just want the logging
+              isBackdatedBene(rifRecordEvent);
             }
             tweakIfBeneficiary(entityManager, loadedBatchBuilder, rifRecordEvent);
             entityManager.persist(record);
           } else if (rifRecordEvent.getRecordAction().equals(RecordAction.UPDATE)) {
             loadAction = LoadAction.UPDATED;
             // Skip this record if the year is not 2022 and its an update.
-            if (isBackdatedBene(rifRecordEvent)) {
+            if (!isSyntheticData && isBackdatedBene(rifRecordEvent)) {
```

Comment:
I think with the logging moved into the isBackDatedBene() method, this will begin printing a message that was not there before and could be confusing since this is the UPDATE case and the logging references inserts.

Reply:
reverted logging back to orig.